Source/swiftlint/Commands/AutoCorrectCommand.swift

```diff
@@ -29,7 +40,8 @@ struct AutoCorrectCommand: CommandProtocol {
             }
             if options.format {
                 let formattedContents = linter.file.format(trimmingTrailingWhitespace: true,
-                                                           useTabs: options.useTabs, indentWidth: 4)
+                                                           useTabs: useTabs,
```

Comment:
Currently `useTabs` is based entirely on your `Configuration` - but for backwards compatibility, I'm guessing we would want to use the command line option if it is provided. Correct? If that is the case, should I add a deprecation warning to the option?

Reply:
Correct.